modmail/bot.py

```diff
@@ -1,15 +1,16 @@
 import asyncio
 import logging
 import signal
+import sys
 import typing as t
-from typing import Any
 
 import arrow
 import discord
 from aiohttp import ClientSession
 from discord import Activity, AllowedMentions, Intents
 from discord.client import _cleanup_loop
 from discord.ext import commands
+from tortoise import BaseDBAsyncClient, Tortoise
```

Comment:
perhaps just `import tortoise`?

Trying to move somewhat away from `from` imports.

Reply:
Why?

Reply:
clutters the namespace and globals

Reply:
Same stands for `from discord import Activity, ...`. And the class names which I imported from tortoise are self-explanatory that they belong to the DB orm, `tortoise`.

Reply:
> Same stands for from discord import Activity, ...


Just because we've done it before doesn't mean we should continue to do it.